packages/vhd-lib/src/createVhdStreamWithLength.integ.spec.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-env jest */
+
+import execa from 'execa'
+import fs from 'fs-extra'
+import rimraf from 'rimraf'
+import getStream from 'get-stream'
+import tmp from 'tmp'
+import { createReadStream, createWriteStream } from 'fs-promise'
+import { fromEvent, pFromCallback } from 'promise-toolbox'
+
+import { createVhdStreamWithLength } from '.'
+import { FOOTER_SIZE } from './_constants'
+
+let tempDir = null
+
+beforeEach(async () => {
+  tempDir = await pFromCallback(cb => tmp.dir(cb))
+})
+
+afterEach(async () => {
+  await pFromCallback(cb => rimraf(tempDir, cb))
+})
+
+async function convertFromRawToVhd(rawName, vhdName) {
+  await execa('qemu-img', ['convert', '-f', 'raw', '-Ovpc', rawName, vhdName])
```

Comment:
I would love to avoid as many env deps as possible…

Do you think it would be possible (not to difficult) to remove this one?